lib/facility.rb

```diff
@@ -1,14 +1,60 @@
 class Facility
   attr_reader :name, :address, :phone, :services
+  attr_accessor :collected_fees, :register_vehicles
```

Comment:
We don't need these in attr_accessors. Adding a method to attr_accessor allows other classes to change these attributes, so we don't want to expose that functionality unless we absolutely have to. These should just be attr_readers.